src/eduid_webapp/actions/tests/test_app.py

```diff
@@ -154,8 +153,8 @@ def test_get_config_fails(self):
                     self.prepare_session(sess, action_error=True)
                     response = client.get('/config')
                     self.assertEqual(response.status_code, 200)
-                    data = cbor.loads(response.data)[0]
-                    self.assertEquals(data['message'], 'test error')
+                    data = json.loads(response.data.decode('ascii'))
```

Comment:
Should not decode('utf-8') work as json by default is utf-8?

Reply:
Yes you are right